### PR TITLE
updated crcr entries

### PR DESCRIPTION
**Pull request by Sowmyak442**

### CODE REVIEW

**Review by michael-fleming — DISMISSED** [review]

File: monitoring/FNL-Monitoring-List.csv [hunk 1]
```diff
@@ -565,4 +565,8 @@ $http.post('https://bento-tools.org/v1/public-graphql/',
     assert.ok('numberOfPrograms' in body.data);
 
   }
-);",,FALSE,C06AYDW6S2Z,,
\ No newline at end of file
+);",,FALSE,C06AYDW6S2Z,,
+Computation Resources for Cancer Research,CRCR,DOERC,dev,Portal,https://computational-dev.cancer.gov/,,,TRUE,C071J4NEJJ3,,
```
Comment (saman-nih) [inline monitoring/FNL-Monitoring-List.csv:569]: should we add DEV and QA? We will have CD, which will result in many notifications that we should ignore

Comment (saman-nih) [reply monitoring/FNL-Monitoring-List.csv:569]: which slack channel is C071J5E4RU7?

**Review by saman-nih — APPROVED** [review]

**Review by kiran1942 — APPROVED** [review]

**Comment by kiran1942** [comment]
LGTM

**Review by vdonkor — APPROVED** [review]
LGTM

**Review by jw34 — APPROVED** [review]
LGTM

**Pull request merged**